### PR TITLE
[CodeClean] element type classification

### PR DESCRIPTION
**Pull request by jaeyun-jung**
element name in nnstreamer and general classification of element.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: ext/nnstreamer/android_source/gstamcsrc.c [hunk 1]
```diff
@@ -238,12 +238,13 @@ gst_wrapped_buf_unref (GstWrappedBuf * self)
  * @brief define boxed type for a wrapped buf
  */
 G_DEFINE_BOXED_TYPE (GstWrappedBuf, gst_wrapped_buf,
-    gst_wrapped_buf_ref, gst_wrapped_buf_unref)
+    gst_wrapped_buf_ref, gst_wrapped_buf_unref);
```
Comment (dongju-chae) [inline ext/nnstreamer/android_source/gstamcsrc.c:241]: Oh.. how this was not checked until now.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Comment by jaeyun-jung** [comment]
2020-12-29 PR updated (avoid assertion / prevent ci error in src-iio)

**Review by wooksong — APPROVED** [review]
LGTM!

**Pull request merged**